src/modules/wallet/types.ts

```diff
@@ -24,6 +24,6 @@ export interface EthereumWindow {
     _metamask: { isApproved: () => Promise<boolean> }
     isApproved: () => Promise<boolean>
 
-    enable: () => Promise<string[]>
+    enable?: () => Promise<string[]>
```

Comment:
this can be undefined?

Reply:
i dunno, but the linter was complaining about this line: https://github.com/decentraland/decentraland-dapps/blob/master/src/modules/wallet/utils.ts#L56

saying that the `typeof ethereum.enable === 'function'` was unnecessary because it was typed as a non-optional function

If we remove that part we can switch this back to be a required prop